src/ledger-transport-middleware.ts

```diff
@@ -60,9 +58,6 @@ export class LedgerTransportMiddleware implements TransportMiddleware {
    * @returns An generic interface for communicating with a Ledger hardware wallet to perform operation.
    */
   getEthApp(): MetaMaskLedgerHwAppEth {
-    if (!this.#app) {
```

Comment:
The comment above is now inaccurate: "it create a new eth app instance if not exist."

We should fix that in a follow up PR